### PR TITLE
[RDY] Default mouse grab

### PR DESCRIPTION
**Pull request by mugmuggy**
Enhancement from #1279 to put capturing mouse as default option

### CODE REVIEW

File: CorsixTH/Lua/config_finder.lua [hunk 1]
```diff
@@ -255,6 +256,11 @@ if needs_rewrite then
 'prevent_edge_scrolling = ' .. tostring(config_values.prevent_edge_scrolling) .. '\n' .. [=[
 
 -------------------------------------------------------------------------------------------------------------------------
+-- Capture mouse: By default enabled (capture mouse = true).
+-- ]=] .. '\n' ..
+'capture_mouse = ' .. tostring(config_values.prevent_edge_scrolling) .. '\n' .. [=[
```
Comment (TheCycoONE) [inline CorsixTH/Lua/config_finder.lua]: Too much copy/paste... `config_values.capture_mouse`

Comment (TheCycoONE) [reply CorsixTH/Lua/config_finder.lua]: Maybe mention ctrl-f10 here too. e.g. something like "Initial mouse capture state when the game starts. To toggle in game press ctrl-f10."

Comment (mugmuggy) [reply CorsixTH/Lua/config_finder.lua]: dang it, I was testing it, and did undo but must have did it on my repo instead of my test folder

**Comment by TheCycoONE** [comment]
Merged.

**Pull request closed without merging**